sections/algorithmic_components.tex

```diff
@@ -95,48 +95,11 @@ \subsection{Artifact Detection}
 
 \subsubsection{Cosmic Ray Identification}
 \label{sec:acCosmicRayDetection}
-Cosmic rays will be identified using a Laplacian edge detection algorithm \citep{2001PASP..113.1420V}.  Laplacian edge detection involves convolving the image (I) with a smoothing function (f) tuned to the size of the expected edge width.
-\[
-L = \bigtriangledown^2f\ast I
-\]
-In the case of cosmic rays, this implies a very sharp edge.  A natural choice is a Gaussian with small $\sigma$.  The discrete kernel chosen in \citep{2001PASP..113.1420V} is
-\begin{align}
-\bigtriangledown^2 = \frac{1}{4}\left( \begin{array}{ccc}
-0 & -1 & 0 \\
--1 & 4 & -1 \\
-0 & -1 & 0 \end{array} \right)
-\end{align}
-If just used on the native image, this filter would attenuate the signal from CRs that effect multiple
-adjacent pixels.  So, the image is subsampled by some factor $f_s$.  Before downsampling to the native resolution, the negative pixel values are clamped to zero.  The resultant image in native pixelization will be referred to as $L^+$.
-
-We construct a SNR image by dividing the Laplacian image by the noise in the image and adjusting by the subsampling factor.
-\[
-S = \frac{L^+}{f_s N}
-\]
+The need for a morphological cosmic ray rejection algorithm is motivated on multiple fronts.  Firstly, the science pipelines are explicitly required to run on visits that are taken in the traditional way, i.e. one single continuous integration, and in a series of snaps where multiple exposures are taken in series and then aggregated in ISR.  Even when the pipelines have the luxury of having multiple exposures taken in quick succession at the same pointing, we cannot simply reject anything in the difference of the snaps as a cosmic ray since we expect to be able to utilize measurements on the snap differences to identify very rapidly varying objects.
```

Comment:
If you update your `lsst-texmf` (see lsst/lsst-texmf#44) you can use `\ossreq{0288}` and you'll get a little requirement number in the margin.